python/api/user/serializers.py

```diff
@@ -65,14 +66,22 @@ class GroupDetailSerializer(GroupSerializer):
 
 
 class UserSerializer(serializers.Serializer):
-    username = serializers.CharField()
+    username = serializers.CharField(max_length=MAX_NAME_LENGTH)
     password = serializers.CharField(write_only=True)
     url = hlink('user-details', 'username', 'username')
     change_group = hlink('add-user-group', 'username', 'username')
     change_password = hlink('user-passwd', 'username', 'username')
     change_role = hlink('change-user-role', 'username', 'username')
     is_superuser = serializers.BooleanField(required=False)
 
+    def validate_username(self, name):
+        if len(name) > MAX_NAME_LENGTH:
+            raise AdcmEx(
+                'LONG_NAME',
+                f'Username is too long, up to {MAX_NAME_LENGTH} symbols required',
+            )
```

Comment:
Need to make this error readable at UI
<img width="1440" alt="Screenshot 2021-07-27 at 11 48 34" src="https://user-images.githubusercontent.com/40051639/127126082-648d980f-8cdf-408b-a657-8c33bc930a87.png">

Reply:
Fixed in last commit